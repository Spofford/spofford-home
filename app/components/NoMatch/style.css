.no-match .copy-container {
  text-align:center;
}
