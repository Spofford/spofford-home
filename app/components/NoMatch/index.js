import React from "react"
import cssModules from "react-css-modules"
import style from "./style.css"
import { connect } from "react-redux"
import { default as Header } from "../Header"
import { Link } from 'react-router-dom'

export class NoMatch extends React.Component {
  render() {
    return (
      <div className="no-match">
        <Header />
        <div className="body-container">
          <div className="copy-container">
          <h2>Nothing to see here.</h2>
          <p>Check the address, <Link to="/">return to home</Link> or <Link to="/login">login</Link>.</p>
          </div>
        </div>
      </div>
    )
  }
}

export default NoMatch
