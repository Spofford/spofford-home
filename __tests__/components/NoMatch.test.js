import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';

import NoMatch from '../../app/components/NoMatch';

describe('<NoMatch />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<NoMatch />);
      // const component = wrapper.dive();

      expect(toJson(wrapper)).toMatchSnapshot();
    });

    // test to render model from contentful
  });
});
