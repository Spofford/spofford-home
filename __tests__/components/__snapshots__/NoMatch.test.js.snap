// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> render() renders the component 1`] = `
<div
  className="no-match"
>
  <Connect(Header) />
  <div
    className="body-container"
  >
    <div
      className="copy-container"
    >
      <h2>
        Nothing to see here.
      </h2>
      <p>
        Check the address, 
        <Link
          replace={false}
          to="/"
        >
          return to home
        </Link>
         or 
        <Link
          replace={false}
          to="/login"
        >
          login
        </Link>
        .
      </p>
    </div>
  </div>
</div>
`;
